README.md

```diff
@@ -117,3 +117,16 @@ issues created.
 
 Details on the request and expected response are [available](docs/generic\_webhook.md).
 
+
+## Cleanup script
+
+A python script is provided in scripts/cleanup.py and in a docker container. It can be configured
+using the commandline options to filter rageshakes in a given date range. It is expected to be
+run as a cronjob on a daily basis.
+
+The script is idempotent enough to be re-runnable on a more frequent basis if required. If a cronjob
+cannot be configured, it can be run in a continual mode with something like `--repeat-delay-hours 24`
+to repeat running after approximately 24 hours.
+
+Be aware that each run will scan all files between --max-days and the smallest retention period set,
+ensuring we always have up to date information, however this may be an intensive process.
```

Comment:
I'm not quite sure what this is trying to tell me. I think it's something like this:

```suggestion
Note that this script will scan all logs older than the smallest configured retention period (up to `--max-days`).
This can be an IO-intensive process.
```